.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Although you don't currently have a `test. md` file please make sure to add the file name along with `node_modules` in the `.gitignore` here.
- [ ] I'd also suggest you stay consistent and follow [best practices](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53) when making commits as I noticed some of your commit messages are a bit vague. You can use verbs like "Add", "Update", "Fix" etc...